client/state/stats/lists/utils.js

```diff
@@ -620,51 +675,11 @@ export const normalizers = {
 	},
 
 	statsVisits( payload ) {
-		if ( ! payload || ! payload.data ) {
-			return [];
-		}
-
-		const attributes = [ 'visits', 'likes', 'visitors', 'comments', 'posts' ];
-
-		return payload.data.map( function( record ) {
-			// Initialize data
-			const dataRecord = attributes.reduce( ( memo, attribute ) => {
-				memo[ attribute ] = null;
-				return memo;
-			}, {} );
-
-			// Fill Field Values
-			record.forEach( function( value, i ) {
-				// Remove W from weeks
-				if ( 'period' === payload.fields[ i ] ) {
-					value = value.replace( /W/g, '-' );
-				}
-				dataRecord[ payload.fields[ i ] ] = value;
-			} );
-
-			dataRecord.labelDay = '';
-			dataRecord.labelWeek = '';
-			dataRecord.labelMonth = '';
-			dataRecord.labelYear = '';
-			dataRecord.classNames = [];
-
-			if ( dataRecord.period ) {
-				const date = moment( dataRecord.period, 'YYYY-MM-DD' ).locale( 'en' );
-				const localizedDate = moment( dataRecord.period, 'YYYY-MM-DD' );
-				if ( date.isValid() ) {
-					const dayOfWeek = date.toDate().getDay();
-					if ( ( 'day' === payload.unit ) && ( ( 6 === dayOfWeek ) || ( 0 === dayOfWeek ) ) ) {
-						dataRecord.classNames.push( 'is-weekend' );
-					}
-					dataRecord.labelDay = localizedDate.format( 'MMM D' );
-					dataRecord.labelWeek = localizedDate.format( 'MMM D' );
-					dataRecord.labelMonth = localizedDate.format( 'MMM' );
-					dataRecord.labelYear = localizedDate.format( 'YYYY' );
-				}
-			}
+		return parseChartData( payload, [ 'visits', 'likes', 'visitors', 'comments', 'posts' ] );
+	},
 
-			return dataRecord;
-		} );
+	statsOrders( payload ) {
+		return parseChartData( payload );
```

Comment:
This returns `null` however `return parseChartData( payload.orders );` works as the `data` object needed here (https://github.com/Automattic/wp-calypso/pull/14313/files#diff-6da59aec4d5693709df18761a243d4beR117) is in `payload.orders.data`.